Resources/Prototypes/_Sunrise/Entities/Objects/Clothing/Pouches/pouches.yml

```
@@ -0,0 +1,87 @@
+    quickEquip: false
+    slots:
+    - pocket
+
+#Explosives
+- type: entity
+  parent: [Pouch, ContentsExplosionResistanceBase]
+  id: PouchExplosive
+  name: explosive pouch
+  description: It can carry grenades, plastic explosives, mine boxes, and other explosives.
+  components:
+  - type: Sprite
```

YAMLlint (1.37.1)

[error] wrong indentation: expected at least 3

(indentation)